cmd/eksctl-anywhere/cmd/options.go

```diff
@@ -137,7 +138,18 @@ func newClusterSpec(options clusterOptions) (*cluster.Spec, error) {
 	}
 
 	if clusterSpec.Cluster.IsManaged() && options.managementKubeconfig == "" {
-		options.managementKubeconfig = kubeconfig.FromEnvironment()
+		envKubeconfig := kubeconfig.FromEnvironment()
+		if envKubeconfig != "" {
+			options.managementKubeconfig = envKubeconfig
+		} else {
+			// check if kubeconfig for management cluster exists locally
+			managementKubeconfigPath := kubeconfig.FromClusterName(clusterSpec.Cluster.Spec.ManagementCluster.Name)
+			if validations.FileExistsAndIsNotEmpty(managementKubeconfigPath) {
+				options.managementKubeconfig = managementKubeconfigPath
+			} else {
+				return nil, fmt.Errorf("management kubeconfig file not found, must be present for workload cluster operations")
+			}
+		}
 	}
 
 	if options.managementKubeconfig != "" {
```

Comment:
If this is still empty, then the management cluster will not be set. Is that okay? Should the below logic go under the above if condition instead as ManagementCluster will only be set for managed workload clusters anyways?